packages/strapi-helper-plugin/lib/src/contexts/GlobalContext.js

```diff
@@ -0,0 +1,31 @@
+import React, { createContext, useContext } from 'react';
+import PropTypes from 'prop-types';
+
+const GlobalContext = createContext({});
+
+const GlobalContextProvider = ({ children, ...rest }) => {
+  return (
+    <GlobalContext.Provider value={rest}>{children}</GlobalContext.Provider>
+  );
+};
+
+const useGlobalContext = () => useContext(GlobalContext);
+
+GlobalContextProvider.defaultProps = {
+  currentEnvironment: 'development',
+  disableGlobalOverlayBlocker: () => {},
+  emitEvent: () => {},
+  enableGlobalOverlayBlocker: () => {},
+  plugins: {},
+  updatePlugin: () => {},
```

Comment:
You declared `plugins` and `updatePlugin` in the defaultProps but not in the propTypes